tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/LargePayload/WindowsNamedPipeLargePayloadTests.cs

```diff
@@ -21,13 +21,14 @@ public WindowsNamedPipeLargePayloadTests(ITestOutputHelper output)
         /// <summary>
         /// To be enabled when Windows Named Pipes is available in the MockTracerAgent
         /// </summary>
-        [SkippableFact(Skip = "Windows named pipes are not yet supported in the MockTracerAgent")]
+        [SkippableTheory(Skip = "Windows named pipes are not yet supported in the MockTracerAgent")]
```

Comment:
waaaaait, yes they are 😅 I guess we need to re-enable this test (later)

Reply:
I just changed the test type, it has been like this.

we can see later.